src/stores/cluster/detail.js

```diff
@@ -40,13 +40,16 @@ export default class ClusterDetailStore extends Store {
 
   @observable selectNodeStatus = '';
 
+  @observable env = '';
+
   @action
   fetch = async clusterId => {
     this.isLoading = true;
     const result = await this.request.get(`clusters`, {
       cluster_id: clusterId
     });
     this.cluster = _.get(result, 'cluster_set[0]', {});
+    this.setEnv();
```

Comment:
Dont put `setEnv` here, fetch action just do one thing!

Reply:
need to set env each time on fetch

Reply:
You can format cluster env when rendered in certain component, not in this method